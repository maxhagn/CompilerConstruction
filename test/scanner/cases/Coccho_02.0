35end
$f
$F
2842374