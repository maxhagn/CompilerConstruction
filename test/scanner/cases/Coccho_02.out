num 35
end
num 15
num 15
num 2842374